tests/test-sessions/test-hazelcast-sessions/src/test/java/org/eclipse/jetty/hazelcast/session/ClusteredLastAccessTimeTest.java

```diff
@@ -0,0 +1,38 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2017 Mort Bay Consulting Pty. Ltd.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.hazelcast.session;
+
+import org.eclipse.jetty.server.session.AbstractClusteredLastAccessTimeTest;
+import org.eclipse.jetty.server.session.SessionDataStoreFactory;
+
+public class ClusteredLastAccessTimeTest
```

Comment:
I guess these tests use  embedded hazelcast. Is there any point in also checking with a "remote" hazelcast instance (eg for serialization problems)?